integration-tests/api/__tests__/admin/product-category.ts

```diff
@@ -774,73 +779,138 @@ medusaIntegrationTestRunner({
 
     describe("POST /admin/product-categories/:id", () => {
       beforeEach(async () => {
-        productCategoryParent = await simpleProductCategoryFactory(
-          dbConnection,
-          {
-            name: "category parent",
-          }
-        )
-
-        productCategory = await simpleProductCategoryFactory(dbConnection, {
-          name: "category-0",
-          parent_category: productCategoryParent,
-          rank: 0,
-        })
-
-        productCategory1 = await simpleProductCategoryFactory(dbConnection, {
-          name: "category-1",
-          parent_category: productCategoryParent,
-          rank: 1,
-        })
-
-        productCategory2 = await simpleProductCategoryFactory(dbConnection, {
-          name: "category-2",
-          parent_category: productCategoryParent,
-          rank: 2,
-        })
-
-        productCategoryChild = await simpleProductCategoryFactory(
-          dbConnection,
-          {
-            name: "category child",
-            parent_category: productCategory,
-            rank: 0,
-          }
-        )
-
-        productCategoryChild0 = await simpleProductCategoryFactory(
-          dbConnection,
-          {
-            name: "category child 0",
-            parent_category: productCategoryChild,
-            rank: 0,
-          }
-        )
-
-        productCategoryChild1 = await simpleProductCategoryFactory(
-          dbConnection,
-          {
-            name: "category child 1",
-            parent_category: productCategoryChild,
-            rank: 1,
-          }
-        )
+        await breaking(
+          async () => {
+            productCategoryParent = await simpleProductCategoryFactory(
```

Comment:
Not for now, but a general comment - It's better to do the seeding through the HTTP API as well so we don't access internals (in this case, modules) if not absolutely necessary